packages/amplify-provider-awscloudformation/src/utils/consolidate-apigw-policies.ts

```diff
@@ -304,3 +304,7 @@ export async function loadApiCliInputs(context: $TSContext, resourceName: string
 
   return stateManager.getResourceInputsJson(projectRoot, AmplifyCategories.API, resourceName, { throwIfNotExist: false });
 }
+
+const appendToUrlPath = (path: string, postfix: string) => {
```

Comment:
if not a huge lift Is it possible to move this function into a common library  like path-utils in amplify-core.